mmap.c

```diff
@@ -54,10 +46,42 @@ int main()
 	// When in capability mode the capability of the coresponding capability register is used, if you try to access a0 that would use the capability inside of ca0
 	//
 	code = cheri_flags_set(code, 0x0001);
+
+}
+
+// The hybrid function works because the DDC can be set to the stack pointer and used as a referance 
+// This because when we are in hybrid mode the DDC's address is what is used, and here we have it set to the stack pointer.
+// This means that when we want to store a value on the stack we can just do `csc cra, 16(cnull)`.
+// The calculateion is dcone like this: `ddc.address + cnull_value + 16`.
```

Comment:
Couple of typos :)

Reply:
Fixed in: https://github.com/capablevms/cheri-examples/commit/a37a4e1ce2a506b587a3a5e8c9b4243689576941